OutOfSchool/OutOfSchool.ExternalFileStore.Impl/NotificationImplementations/MinioRedisNotificationBridge.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.Extensions.Logging;
+using Minio;
+using Minio.DataModel.Args;
+using Minio.DataModel.Notification;
+using OutOfSchool.ExternalFileStore.Models;
+using OutOfSchool.ExternalFileStore.NotificationInterfaces;
+using StackExchange.Redis;
+using System.Text.Json;
+
+namespace OutOfSchool.ExternalFileStore.NotificationImplementations;
+public class MinioRedisNotificationBridge : IDisposable
+{
+    private readonly MinioClient _minioClient;
```

Comment:
If we keep this implementation - do not use Minio directly. There is a wrapper in our project. Check Implementation project Extensions namespace.